src/Domain/UrlNormalizer/BitbucketUrlNormalizer.php

```diff
@@ -29,7 +29,9 @@ private function normalizePath( string $path, bool $toRawUrl ): string {
 			$urlParts[5] = 'raw';
 		}
 
-		$urlParts[6] = ( $urlParts[6] ?? '' ) === '' ? 'README.md' : $urlParts[6];
+		if ( !isset( $urlParts[6] ) || $urlParts[6] === '' ) {
```

Comment:
What is the issue with the old form?

Reply:
It triggered [`NullArrayOffset`](https://psalm.dev/docs/running_psalm/issues/NullArrayOffset/) when I run locally, which is weird since it should have been handled. I'll rerun again with the old code to see if it passes the CI

Reply:
Weird. Maybe this old Psalm version has a bug, or did not yet support `??`. In such cases you can always put the error into the baseline file, though I see we do not have our now-usual baseline file and command setup in this older repo.

But it does not matter. New code also works. Let's not spend more time